code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -294,12 +294,34 @@ obj/item/asteroid/basilisk_hide/New()
 	icon = 'icons/obj/food.dmi'
 	icon_state = "boiledrorocore"
 	var/inert = 0
+	var/timer = 60
 
 /obj/item/asteroid/hivelord_core/New()
 	..()
-	spawn(1200)
+	create_reagents(5)
+	processing_objects.Add(src)
+
+/obj/item/asteroid/hivelord_core/Destroy()
+	processing_objects.Remove(src)
+	..()
+
+/obj/item/asteroid/hivelord_core/process()
```

Comment:
Doing it like this is bad because it makes the decay speed entirely dependent on the process's tick rate. Make it work with `world.time` timestamps instead.